src/main/resources/assets/modmenu/lang/es_mx.json

```
@@ -0,0 +1,36 @@
+{
+  "modmenu.title": "Mods",
+  "modmenu.loaded": "(%s cargada)",
+  "modmenu.config": "Editar configuración",
+  "modmenu.modsFolder": "Abrir carpeta de mods",
+  "modmenu.configsFolder": "Abrir carpeta de configuraciones",
+  "modmenu.configure": "Configurar...",
+  "modmenu.modIdToolTip": "ID del Mod: %s",
+  "modmenu.website": "Sitio web",
+  "modmenu.issues": "Problemas",
+  "modmenu.authorPrefix": "de %s",
+  "modmenu.search": "Buscar mods",
+  "modmenu.clientsideOnly": "Cliente",
+  "modmenu.deprecated": "Obsoleta",
+  "modmenu.library": "Librería",
+  "modmenu.toggleFilterOptions": "Alternar opciones de filtro",
+  "modmenu.showLibraries": "Librerías: %s",
+  "modmenu.showLibraries.true": "Mostrada",
+  "modmenu.showLibraries.false": "Oculta",
```

Shouldn't those be plural, since it's about `Librerías: %s` in the filters menu?
```suggestion
  "modmenu.showLibraries.true": "Mostradas",
  "modmenu.showLibraries.false": "Ocultas",
```
Or maybe even in infinitive `Mostrar`/`Ocultar`?
```suggestion
  "modmenu.showLibraries.true": "Mostrar",
  "modmenu.showLibraries.false": "Ocultar",
```

---

`Mostrar/Ocultar` sounds better here.